frontend/utils/index.js

```diff
@@ -0,0 +1,24 @@
+import { clsx } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs) {
+  return twMerge(clsx(inputs));
+}
+
+export const fetchStrapiData = async (endpoint, params = {}) => {
+  const queryString = new URLSearchParams(params).toString();
+  const url = `${process.env.NEXT_PUBLIC_STRAPI_API_URL}${endpoint}?${queryString}`;
```

Comment:
@navins94 This will not work on test and prod server. Use the same approach as in TDT.